### ISSUE TITLE
Link to picture not removed in folder view after deletion with trash icon

### ISSUE THREAD

**Issue by nielscastien** [nextcloud/server#9722]
### Steps to reproduce
1. Open picture in full view mode from any folder in Nextcloud
2. Use trash icon in right corner to delete picture
3. Return to folder by using exit icon

### Expected behaviour
When back in the folder overview, the link to the picture should be gone

### Actual behaviour
Link to picture is not deleted. Only after refreshing the folder view the link to the picture disappears. 

### Server configuration

**Operating system**: Ubuntu Linux 16.0.4.4 LTS

**Web server:** Apache 2.4.18

**Database:** MySQL 5.7.22

**PHP version:** 7.0.30

**Nextcloud version:** 13.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated

**Where did you install Nextcloud from:** source

**Comment by MorrisJobke** [comment]
This might be fixed by #2742 or we need to implement a forced refresh of the files list from apps.

@juliushaertl @skjnldsv @danxuliu Is it worth the time to sixths if we want to redo the files app with 15 anyways?

**Comment by juliushaertl** [comment]
Right now the big image view is done by the gallery app. I'm not sure how much effort this would be to push changes from there to the files app. So I'd say we should have a look at this for 15.

Having some live update mechanism is something i might look into for 15. It would be nice to have something generic here, that other apps could use as well. I'll open an issue for that later.

**Comment by skjnldsv** [comment]
Agree, not sure it's worth it since we'll work ont he new files app :)

**Comment by MorrisJobke** [comment]
Okay - let's focus on #2742 for 15 and then this one should be not an issue anymore. Thus I will close this for now.
